test/unit/specs/core/createCookieProxy.spec.js

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import createCookieProxy from "../../../../src/core/createCookieProxy";
+import cookie from "../../../../src/utils/cookie";
+
+const COOKIE_NAME = "testcookie";
+const COOKIE_EXPIRES = 180;
+const removeCookie = name => {
+  document.cookie = `${name}=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;`;
+};
+
+const removeAllCookies = () => {
+  const cookies = document.cookie.split(";");
+  cookies.forEach(cookieName => {
+    removeCookie(cookieName.split("=")[0]);
+  });
+};
+
+describe("createCookie", () => {
```

Comment:
should describe `createCookieProxy`

Reply:
Fixed this is a later commit. Sorry for pushing commits while you were reviewing.